region/lowernorfair/east.json

```diff
@@ -814,6 +845,38 @@
         {
           "from": 3,
           "to": [
+            {
+              "id": 1,
```

Comment:
Right, but it also takes additional heat frames, and there are many ways to continue up to 4, so it would multiply the amount of strats significantly. Since there are already 1 -> 4 strats, I thought it would be fine for now to just add the strats going up to 1. We could add direct strats to 4 (or add another junction) later if we want to refine this.